frame/im-online/src/tests.rs

```diff
@@ -174,8 +174,7 @@ fn late_heartbeat_should_fail() {
 	new_test_ext().execute_with(|| {
 		advance_session();
 		// given
-		VALIDATORS.with(|l| *l.borrow_mut() = Some(vec![1, 2, 4, 4, 5, 6]));
-		assert_eq!(Session::validators(), Vec::<u64>::new());
+		VALIDATORS.with(|l| *l.borrow_mut() = Some(vec![1, 2, 3, 4, 5, 6]));		assert_eq!(Session::validators(), Vec::<u64>::new());
```

Comment:
```suggestion
		VALIDATORS.with(|l| *l.borrow_mut() = Some(vec![1, 2, 3, 4, 5, 6]));		
		assert_eq!(Session::validators(), Vec::<u64>::new());
```

or maybe github got confused here?